app/javascript/components/student/iteration-page/TestsInformation.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { Request, usePaginatedRequestQuery } from '../../../hooks/request-query'
+import { FetchingBoundary } from '../../FetchingBoundary'
+import { ResultsZone } from '../../ResultsZone'
+import { useIsMounted } from 'use-is-mounted'
+import { TestRunSummary } from '../../editor/TestRunSummary'
+import { TestRun } from '../../editor/types'
+
+const DEFAULT_ERROR = new Error('Unable to fetch test run')
+
+type APIResponse = {
+  testRun: TestRun
+}
+
+export const TestsInformation = ({
```

Comment:
This looks similar to `<TestsFailedModal />` but I decided not to DRY them up yet.

Reply:
Yeah no need to do that I think.